Index: phpmyadmin/libraries/vendor_config.php
===================================================================
--- phpmyadmin.orig/libraries/vendor_config.php
+++ phpmyadmin/libraries/vendor_config.php
@@ -22,19 +22,19 @@ define('AUTOLOAD_FILE', './vendor/autolo
 /**
  * Directory where cache files are stored.
  */
-define('TEMP_DIR', './tmp/');
+define('TEMP_DIR', '/tmp/phpmyadmin/');
 
 /**
  * Path to changelog file, can be gzip compressed. Useful when you want to
  * have documentation somewhere else, eg. /usr/share/doc.
  */
-define('CHANGELOG_FILE', './ChangeLog');
+define('CHANGELOG_FILE', '/usr/share/doc/phpmyadmin/changelog.gz');
 
 /**
  * Path to license file. Useful when you want to have documentation somewhere
  * else, eg. /usr/share/doc.
  */
-define('LICENSE_FILE', './LICENSE');
+define('LICENSE_FILE', '/usr/share/doc/phpmyadmin/copyright');
 
 /**
  * Directory where SQL scripts to create/upgrade configuration storage reside.
@@ -66,7 +66,7 @@ define('CUSTOM_FOOTER_FILE', CONFIG_DIR
 /**
  * Default value for check for version upgrades.
  */
-define('VERSION_CHECK_DEFAULT', true);
+define('VERSION_CHECK_DEFAULT', false);
 
 /**
  * Path to files with compiled locales (*.mo)
